incubator/kylo/values.yaml

```diff
@@ -0,0 +1,418 @@
+# Default values for Kylo.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+## Kubernetes configuration
+## For minikube, set this to NodePort, elsewhere use LoadBalancer
+## Use ClusterIP if your setup includes ingress controller
+serviceType: LoadBalancer
+
+## Probes
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes)
+probe:
+  initialDelaySeconds: 30
+  timeoutSeconds: 15
+
+# Sidecar that is used to tail NiFi logs. Allows for the set up containers for tailing:
+# nifi-app
+# nifi-user
+# nifi-bootstrap
+sidecar:
+  image: busybox
+
+## Two Way SSL
+## ref:  https://github.com/apache/nifi/tree/master/nifi-docker/dockerhub#standalone-instance-two-way-ssl
+tls:
+  enabled: false
+  keystorePath: /opt/certs/keystore.jks
+  keystoreType: JKS
+  keystorePassword: QKZv1hSWAFQYZ+WU1jjF5ank+l4igeOfQRp+OSbkkrs
+  truststorePath: /opt/certs/truststore.jks
+  truststore_Password: rHkWR1gDNW3R9hgbeRsT3OM3Ue0zwGtQqcFKJD2EXWE
+  truststore_Type: JKS
+  initialAdminIdentity: 'CN=Random User, O=Apache, OU=NiFi, C=US'
+
+## LDAP
+## ref: https://github.com/apache/nifi/tree/master/nifi-docker/dockerhub#standalone-instance-ldap
+ldap:
+  enabled: false
+  authenticationStrategy: 'SIMPLE'
+  manager_DN: 'cn=admin,dc=example,dc=org'
+  manager_Password: 'password'
+  userSearchBase: 'dc=example,dc=org'
+  userSearchFilter: 'cn={0}'
+  identityStrategy: 'USE_DN'
+  url: 'ldap://ldap:389'
+  tlsKeystore: ''
+  tlsKeystorePassword: ''
+  tlsKeystoreType: ''
+  tlsTruststore: ''
+  tlsTruststorePassword: ''
+  tlsTruststoreType: ''
+  tlsProtocol: ''
+
+## ActiveMQ configuration
+activemq:
+  image:
+    repository: govcloud/docker-kylo
+    pullPolicy: Always
```

Comment:
Can the `pullPolicy` be `IfNotPresent` by default. This applies to all places images are fetched and have an explicit version. See the guidelines for more detail on this... https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#configuration